src/FragmentResolver.ts

```diff
@@ -275,19 +273,7 @@ export class FragmentResolver {
             complete();
         };
 
-        const refetchSubscription = this.executeFetcher(
-            taggedNode,
-            fetchVariables,
-            options,
-            observer,
-            onNext,
-        );
-
-        return {
-            dispose: (): void => {
-                refetchSubscription && refetchSubscription.unsubscribe();
-            },
-        };
+        return this.executeFetcher(taggedNode, fetchVariables, options, observer, onNext);
```

Comment:
Maybe add some tests to this?

Reply:
I think this test case is enough
https://github.com/relay-tools/relay-hooks/blob/master/__tests__/ReactRelayRefetchContainer-test.tsx#L916